src/main/scala/Commands/About/Help.scala

```diff
@@ -0,0 +1,31 @@
+package Commands.About
+
+import Commands.{Command, SubCommand}
+import cats.effect.{IO, Resource}
+import doobie.hikari.HikariTransactor
+import doobie.util.transactor.Transactor
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent
+
+object Help extends Command with SubCommand{
+  override val aliases: Seq[String] = Seq("help")
+  override val name: String = "Help"
+
+  override def getSubCommand(event: MessageReceivedEvent): SubCommand = {
+    event.getMessage.getContentRaw.split("\\s+").lift(1) match{
+      case None => Help
+      case Some(value) => {
+        case _ => Help
+      }
+    }
```

Comment:
I don't think there's any Sub Commands planned for !help, so there's no need to pattern match your way to the Help object. I'd just have the method provide the Help command right away.